test/test_cputime_with_memory.py

```diff
@@ -0,0 +1,71 @@
+"""
+Because limiting on windows sets limits on a Job, a thing that wraps
+a process twice, in a manner quite different to Unix systems, we test
+that using both limits does not interfere with
+"""
+import time
+
+from pynisher import CpuTimeoutException, MemoryLimitException, Pynisher, supports
+from pynisher.util import memconvert
+
+import pytest
+
+if not supports("memory") or not supports("cpu_time"):
+    pytest.skip("Tests specifically for cputime and memory", allow_module_level=True)
+
+
+def usememory(x: int) -> int:
+    """Use a certain amount of memory in B"""
+    bytearray(int(x))
+    return x
+
+
+def cpu_busy(execution_time: int) -> float:
```

Comment:
That's roughly the same function as `sleep` in `test_cpu_time.py`, right? Can they be put into a general location?

Reply:
They could,I just made each test file self contained, I can throw them into one file.